e2e/playwright/info-drawer/src/tests/library-properties.spec.ts

```diff
@@ -0,0 +1,270 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, Utils, test, SitesApi, QueriesApi, SITE_VISIBILITY, SITE_ROLES } from '@alfresco/playwright-shared';
+
+test.describe('Library properties', () => {
+  let sitesApi: SitesApi;
+  let queriesApi: QueriesApi;
+  const username = `user1-${Utils.random()}`;
+  const user2 = `user2-${Utils.random()}`;
+  const user3 = `user3-${Utils.random()}`;
+
+  const site = {
+    name: `site1-${Utils.random()}`,
+    id: `site-id-${Utils.random()}`,
+    visibility: SITE_VISIBILITY.MODERATED,
+    description: 'my site description'
+  };
+
+  const siteForUpdate = {
+    name: `site2-${Utils.random()}`,
+    id: `site-id-${Utils.random()}`,
+    visibility: SITE_VISIBILITY.MODERATED,
+    description: 'my initial description'
+  };
+
+  const siteUpdated = {
+    name: `site-for-rename-${Utils.random()}`,
+    visibility: SITE_VISIBILITY.PRIVATE,
+    description: 'new description'
+  };
+
+  const siteDup = `site3-${Utils.random()}`;
+
+  test.beforeEach(async ({ myLibrariesPage, loginPage }) => {
+    try {
+      await loginPage.loginUser({ username, password: username }, { withNavigation: true, waitForLoading: true });
+      await myLibrariesPage.navigate();
+    } catch (error) {
+      console.error(` beforeEach failed: ${error}`);
+    }
+  });
+
+  test.afterEach(async ({ myLibrariesPage }) => {
+    if (await myLibrariesPage.libraryDetails.infoDrawerPanel.isVisible()) {
+      await myLibrariesPage.acaHeader.viewDetails.click();
+    }
+  });
+
+  test.beforeAll(async () => {
+    try {
+      const apiClientFactory = new ApiClientFactory();
+      await apiClientFactory.setUpAcaBackend('admin');
+      await apiClientFactory.createUser({ username: username });
```

Comment:
can be just: 
{ username } 

no need to repeat at both sides when name is the same